h2o-r/tests/runitUtils/matchers.R

```diff
@@ -0,0 +1,10 @@
+library(rlang)
+
+#TODO How to make it visible globally?
+expect_defined <- function (object, info = NULL, label = NULL)
+{
+    act <- quasi_label(enquo(object), label)
+    expect(!is.null(act$val), sprintf("%s is not defined.", act$lab),
+    info = info)
+    invisible(act$val)
+}
```

Comment:
Can remove above and just use `expect_equal`. Also, we should not need new testing functions for the most part. The `testtat` library should have good coverage.